components/cli/vendor/github.com/docker/docker/opts/config.go

```diff
@@ -1,98 +0,0 @@
-package opts
```

Comment:
i traced the sequence of events and it looks like this missing `vendor` change is a result of creating the docker-ce 17.06 branch before the cli was re-vendored. here is the sequence of events:

- May 19: merge moby/moby#33296 to moby/moby which removed `opts/config.go`
- May 19: create docker-ce 17.06 branch
- Jul 7: re-vndr docker/docker to remove `opts/config.go` from `vendor` docker/cli@44ac808